modules/installation/proc-install-rhdh-airgapped-environment-ocp-operator.adoc

```diff
@@ -5,68 +5,159 @@
+----
+* If you want to use an existing target mirror registry, specify it with the `--use_existing_mirror_registry` option. For example:
++
+[source,terminal,subs="+quotes,+attributes"]
+----
 bash prepare-restricted-environment.sh \
    --prod_operator_index "registry.redhat.io/redhat/redhat-operator-index:v{ocp-version}" \
    --prod_operator_package_name "rhdh" \
    --prod_operator_bundle_name "rhdh-operator" \
    --prod_operator_version "v{product-bundle-version}" \
-   --use_existing_mirror_registry "my_registry"
+   --use_existing_mirror_registry "_<my_registry>_"
 ----
 +
 [NOTE]
 ====
 The script can take several minutes to complete as it copies multiple images to the mirror registry.
 ====
+
+.Verification
+* If you are using {ocp-brand-name}, the {product} Operator is in the *Installed Operators* list in the web console.
+* If you are using a supported Kubernetes platform, you can check the list of pods running in the `rhdh-operator` namespace by running the following command in your terminal:
++
+[source,terminal,subs="+quotes,+attributes"]
+----
+kubectl -n rhdh-operator get pods
+----
+
+== Installing {product} on {ocp-short} in a fully disconnected environment with the Operator
+
+If your network has access to the registry through a bastion host or physical disk, you can use the Operator to install {product} by mirroring specified resources and transferring them to your air-gapped environment without any connection to the internet.
+
+.Prerequisites
+
+* You have set up your disconnected environment.
+** You have mirrored all of the required images to disk.
+** You have manually transferred the mirror folder to the network of the disconnected mirror registry.
+** You have mirrored the images from disk to the target mirror registry in your disconnected environment.
+** You have installed the Operator in your disconnected environment.
+* You have set up your workstation.
+** You have an active `oc registry` session to the `registry.redhat.io` {company-name} Ecosystem Catalog. For more information, see link:https://access.redhat.com/RegistryAuthentication[{company-name} Container Registry Authentication].
+** You have installed the `opm` CLI tool. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/cli_tools/opm-cli#olm-about-opm_cli-opm-install[Installing the opm CLI].
+** You have installed Podman 5.3 or later. For more information, see link:https://podman.io/docs/installation[Podman Installation Instructions].
+
+.Procedure
+. Download the mirroring script to disk by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+curl -sSLO https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/release-{product-version}/.rhdh/scripts/prepare-restricted-environment.sh
+----
++
+. Run the mirroring script by running the `bash` command with the appropriate set of options:
++
+[source,terminal,subs="+quotes,+attributes"]
+----
```

Comment:
The command you mentioned (using `--ci-index true`) is for CI testing purposes, and should not be part of the product docs. 

Instead, this is what the end-user needs to run to mirror the images to disk:

```
bash prepare-restricted-environment.sh \
    --to-dir _<my_pulled_image_location>_
```